subsys/net/lib/nrf_cloud/include/nrf_cloud_codec_internal.h

```diff
@@ -171,6 +171,15 @@ int nrf_cloud_shadow_control_process(struct nrf_cloud_obj_shadow_data *const inp
 int nrf_cloud_coap_shadow_default_process(struct nrf_cloud_obj_shadow_data *const input,
 					  struct nrf_cloud_data *const response_out);
 
+/** @brief Get the info sections that are to be sent to the device's shadow on initial connection.
+ * The sections are selected based on the configuration options that set the Kconfig
+ * @kconfig{CONFIG_NRF_CLOUD_SEND_SHADOW_INFO} symbol.
+ *
+ * @retval 0		Success.
+ * @retval -ENODEV	CONFIG_NRF_CLOUD_SEND_SHADOW_INFO is disabled.
+ */
+int nrf_cloud_shadow_info_sections_to_send_get(struct nrf_cloud_device_status *const ds);
```

Comment:
I think this function name would read better if "_to_send" were removed.

Reply:
i agree it would read better, but i'd rather keep the ugly, more specific name.
`nrf_cloud_get_shadow_info_sections_to_send` would be preferred, but alas...

Reply:
ok, I'll give up after this, but here's one more idea: `nrf_cloud_shadow_info_enabled_sections_get`.

Reply:
🤝